src/utils/mod.rs

```diff
@@ -607,3 +607,65 @@ pub fn is_expn_of(cx: &LateContext, mut span: Span, name: &str) -> Option<Span>
         }
     }
 }
+
+// returns index after first CamelCase component of string
```

Comment:
indeed. Can we lint on comments before functions/types/modules that should be doc-comments? Comments don't show up in the ast afaik, so this is probably messy :(